### PR TITLE
Add ./run script

### LINKED ISSUE #6: Add ./run script
- Setup ./run script
- Document properly in README

### PR DESCRIPTION
**Pull request by nottrobin**
This uses the pre-release version 1.1.0 of the run script, and so reviewing this branch is in a way a QA test for https://github.com/canonical-webteam/generator-canonical-webteam/pull/35

Fixes #6

If reviewing this PR goes well, you've basically done half the job of reviewing https://github.com/canonical-webteam/generator-canonical-webteam/pull/35 also - so you may as well go and review that PR too ;)

QA
--

``` bash
./run help
./run  # Check it runs the server on port 8021
./run clean
./run watch
./run test
... etc.
```

### CODE REVIEW

**Comment by nottrobin** [comment]
@bartaz I believe I've fixed the problem you were having. Please try again?

**Comment by bartaz** [comment]
Some issues running node-sass:

```
~/Canonical/snapcraft-flask [nottrobin-6-run-script] $ ./run
yarn install v0.24.4
warning No license field
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
warning Your current version of Yarn is out of date. The latest version is "0.27.5" while you're on "0.24.4".
Done in 7.28s.
yarn run v0.24.4
$ node-sass --include-path node_modules static/sass --output static/css 
/Users/bartaz/Canonical/snapcraft-flask/node_modules/node-sass/lib/binding.js:15
      throw new Error(errors.missingBinary());
      ^

Error: Missing binding /Users/bartaz/Canonical/snapcraft-flask/node_modules/node-sass/vendor/linux-x64-51/binding.node
Node Sass could not find a binding for your current environment: Linux 64-bit with Node.js 7.x

Found bindings for the following environments:
  - OS X 64-bit with Node.js 6.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass --force` to build the binding for your current environment.
    at module.exports (/Users/bartaz/Canonical/snapcraft-flask/node_modules/node-sass/lib/binding.js:15:13)
    at Object.<anonymous> (/Users/bartaz/Canonical/snapcraft-flask/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/bartaz/Canonical/snapcraft-flask/node_modules/node-sass/bin/node-sass:11:10)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by bartaz** [comment]
Running `./run clean` seemed to fix the previous issues, but now `./run` fails differently:

```
~/Canonical/snapcraft-flask [nottrobin-6-run-script] $ ./run
yarn install v0.24.4
warning No license field
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.49s.
yarn run v0.24.4
$ node-sass --include-path node_modules static/sass --output static/css 
Rendering Complete, saving .css file...
Wrote CSS to /Users/bartaz/Canonical/snapcraft-flask/static/css/styles.css
Wrote 1 CSS files to /Users/bartaz/Canonical/snapcraft-flask/static/css
Done in 1.62s.
./run: line 240: type: md5sum: not found
docker: Error response from daemon: driver failed programming external connectivity on endpoint canonical-webteam-snapcraft-flask-b143e1e2-serve (c94c6baaa05a52a86006a148c42ed040827d9538ab1dffe9e06416263652d10a): Error starting userland proxy: Bind for 0.0.0.0:8021 failed: port is already allocated.
```

Seems like port :8021 is already used, but I don't seem to have anything listening on that...

**Comment by nottrobin** [comment]
@bartaz what does `docker ps` show you? You can probably fix it with `docker kill ${container_id}`, but I'd also like to understand how we ended up with a rogue docker image...

**Comment by bartaz** [comment]
@robin `docker ps` is empty

File: README.md [hunk 1]
```diff
@@ -4,39 +4,27 @@ This is the new [Flask](http://flask.pocoo.org/) application that will gradually
...
 ``` bash
-python3 -m venv env3             # Create a python3 virtual environment
-source env3/bin/activate         # Use the python3 environment
-pip install -r requirements.txt  # Install python dependencies
-yarn install                     # Install NPM dependencies into node_modules
+./run
 ```
 
-### Building CSS
+Once the containers are setup, you can visit <http://127.0.0.1:8021> in your browser.
```
Comment (bartaz) [inline README.md]: We changed port to 8022

Comment (bartaz) [inline README.md]: We changed port to 8022

Comment (nottrobin) [reply README.md]: Good catch

**Pull request merged**